packages/core/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@latitude-data/typescript-config/base.json",
   "compilerOptions": {
+    "moduleResolution": "nodenext",
```

Comment:
I needed this to make it work t3-env, default in base is `node`. 😂 Let's hope is fine.